src/Medidata.ZipkinTracer.Core/SpanTracer.cs

```diff
@@ -73,24 +76,37 @@ public virtual void SendServerSpan(Span span)
             spanCollector.Collect(span);
         }
 
-        public virtual Span SendClientSpan(string spanName, string traceId, string parentSpanId, string spanId, string clientServiceName, string path)
+        public virtual Span SendClientSpan(string spanName, string traceId, string parentSpanId, string spanId, Uri remoteUri)
         {
             var newSpan = CreateNewSpan(spanName, traceId, parentSpanId, spanId);
+            var serviceEndpoint = zipkinEndpoint.GetLocalEndpoint(serviceName);
+            var clientServiceName = GetClientServiceName(remoteUri);
 
             var annotation = new Annotation()
             {
-                Host = zipkinEndpoint.GetEndpoint(clientServiceName),
+                Host = serviceEndpoint,
                 Timestamp = GetTimeStamp(),
                 Value = zipkinCoreConstants.CLIENT_SEND
             };
 
             newSpan.Annotations.Add(annotation);
-            AddBinaryAnnotation("http.uri", path, newSpan);
+            AddBinaryAnnotation("http.uri", remoteUri.AbsolutePath, newSpan, serviceEndpoint);
+            AddBinaryAnnotation("sa", "1", newSpan, zipkinEndpoint.GetRemoteEndpoint(remoteUri, clientServiceName));
```

Comment:
this looks broken.. "sa" (and "ca") are binary annotation of type BOOL which is a 0x01 like one bit not the string "1"